src/components/KMultiselect/KMultiselect.vue

```diff
@@ -1034,35 +1054,46 @@ onBeforeUnmount(() => {
 
         :deep(.k-input) {
           &:not([type=checkbox]):not([type=radio]), .form-control:not([type=checkbox]):not([type=radio]) {
-            background-color: transparent !important;
+            background-color: var(--kui-color-background-transparent, $kui-color-background-transparent) !important;
           }
         }
       }
+
+      &.is-toggled {
+        margin-top: var(--kui-space-20, $kui-space-20) !important;
+      }
     }
   }
 
+  .k-multiselect-list {
+    margin: var(--kui-space-0) !important;
```

Comment:
question: why is this `var()` with no fallback?

question: It seems like there is a significant increase in the usage `!important` for styles post tokenization. Is this desired? It seems like we should be trying to use more selective selectors to take precedence instead.